algebra-utils/src/msm/variable_base.rs

```diff
@@ -203,16 +309,22 @@ mod test {
             .collect::<Vec<_>>();
 
         let naive = naive_var_base_msm(g.as_slice(), v.as_slice());
-        let fast = VariableBaseMSM::multi_scalar_mul(g.as_slice(), v.as_slice());
+        let mixed = VariableBaseMSM::multi_scalar_mul_mixed(g.as_slice(), v.as_slice());
         let affine = VariableBaseMSM::multi_scalar_mul_affine(g.as_slice(), v.as_slice());
 
-        assert_eq!(naive, fast);
-        assert_eq!(naive, affine)
+        #[cfg(feature = "gpu")]
```

Comment:
Would be nice to have separate tests for cpu and gpu (even if there will be some code duplication)

Reply:
Done